pkg/fqdn/dnsproxy/proxy.go

```diff
@@ -46,6 +47,10 @@ const (
 	ProxyBindRetryInterval = ProxyBindTimeout / 5
 )
 
+var (
+	rejectReply = dns.RcodeRefused
```

Comment:
The thing, if I change from proxy I need to change the Refused function to be part of the proxy. I'm happy to do that, but want to make sure.

Reply:
Yeah, I also meant that. It could be a helper that take the RCode and the request message, if you felt the hidden state was a little 0_o I'm not too sure that this matters overall... and we can change it when we need to, so feel free to not make it.

Reply:
made this comment, If you're agree I'll squash the commits: 
https://github.com/cilium/cilium/pull/6571/commits/3179e872b6db1af4ee61766776ae67d05099224e

Reply:
Perfect. Thank you!